lua/nvim-tree/notify.lua

```diff
@@ -1,7 +1,12 @@
 local M = {}
 
+local fallbackHandler = function(msg, level, opts)
```

Comment:
```suggestion
local default_handler = function(msg, level, opts)
```

No camel case please.